src/appengine/handlers/cron/project_setup.py

```diff
@@ -137,15 +137,21 @@ def job_name(self, project_name, config_suffix):
 GFT_UBSAN_JOB = JobInfo('googlefuzztest_ubsan_', 'googlefuzztest', 'undefined',
                         ['googlefuzztest', 'engine_ubsan'])
 
+LIBFUZZER_NONE_JOB = JobInfo("libfuzzer_none_", "libfuzzer", "none", [])
```

Comment:
nit: let's `s/none/nosanitizer/` to make it clearer what `none` means to users.

Reply:
Done.